### PR TITLE
Chore/PrivateRouteHome-C2D-16

### PR DESCRIPTION
**Pull request by Alperdec**
Problem
=======
C2D-16 on Jira. 

Solution
========
Added a private route to /home

Change Summary:
---------------
PrivateRoute home, Route to '/' with new component 'Landingview'
and added some sass changes because the hover jitter was annoying
Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
-----------------------

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -1,2 +0,0 @@
-REACT_APP_AUTH0_DOMAIN=
-REACT_APP_AUTH0_CLIENTID=
```
Comment (w-gao) [inline sample.env]: Why is this removed?

**Review by w-gao — CHANGES_REQUESTED** [review]
We should probably keep `sample.env` as a template right? Otherwise, this looks good to me.

**Review by w-gao — APPROVED** [review]
LGTM!

**Review by krishnakpandian — APPROVED** [review]
Everything Looks good and can be merged!

**Comment by w-gao** [comment]
@krishnakpandian Just to confirm, we are doing squash and merge right?

**Comment by krishnakpandian** [comment]
> @krishnakpandian Just to confirm, we are doing squash and merge right?

Yeah it's better if we squash and merge, I just switched it over
